frontend/src/views/WheresMyGene/components/GeneSearchBar/components/SaveExport/csvUtils.ts

```diff
@@ -0,0 +1,239 @@
+        .map((selected) => selected.label)
+        .join(", ") || ""
+    }`,
+  ]);
+
+  // Disease
+  output.push(["# Disease"]);
+  output.push([
+    `${
+      disease_terms
+        ?.filter((option) => {
+          return selectedFilters.diseases.includes(option.id);
+        })
+        .map((selected) => selected.name)
+        .join(", ") || ""
+    }`,
+  ]);
+
+  // Ethnicity
+  output.push(["# Self-Reported Ethnicity"]);
+  output.push([
+    `${
+      self_reported_ethnicity_terms
+        ?.filter((option) => {
+          return selectedFilters.ethnicities.includes(option.id);
+        })
+        .map((selected) => selected.name)
+        .join(", ") || ""
+    }`,
+  ]);
+
+  // Sex
+  output.push(["# Sex"]);
+  output.push([
+    `${
+      sex_terms
+        ?.filter((option) => {
+          return selectedFilters.sexes.includes(option.id);
+        })
+        .map((selected) => selected.name)
+        .join(", ") || ""
+    }`,
+  ]);
+
+  // Organism
+  output.push(["# Organism"]);
+  output.push([
+    `${
+      availableOrganisms.find((organism) => organism.id === selectedOrganismId)
+        ?.name
+    }`,
+  ]);
+
+  // Column Names
+  if (compare) {
+    const compareOptionName = getCompareOptionNameById(compare);
+    output.push([
+      "Tissue",
+      "Cell Type",
+      "Cell Count",
+      "Tissue Composition",
+      compareOptionName,
+      "Gene Symbol",
+      "Expression",
+      "Expression, Scaled",
+      "Number of Cells Expressing Genes",
+    ]);
+  } else {
+    output.push([
+      "Tissue",
+      "Cell Type",
+      "Cell Count",
+      "Tissue Composition",
+      "Gene Symbol",
+      "Expression",
+      "Expression, Scaled",
+      "Number of Cells Expressing Genes",
+    ]);
+  }
+
+  return output;
+}
+
+export function csvGeneExpressionRow({
+  metadata,
+  tissueName,
+  allChartProps,
+  geneName,
+  compare,
+}: {
+  metadata: CsvMetadata;
+  tissueName: string;
+  allChartProps: { [tissue: string]: ChartProps };
+  geneName: string;
+  compare: CompareId | undefined;
+}) {
+  const { total_count, name, compareValueName, viewId } = metadata;
+
+  const geneExpression = allChartProps[tissueName].chartData.find(
+    (value) => value.id === `${viewId}-${geneName}`
+  );
+
+  if (!compare) {
+    return [
+      tissueName,
+      name,
+      total_count,
+      getTissuePercentage(geneExpression),
+      geneName,
+      geneExpression?.meanExpression ?? "",
+      geneExpression?.scaledMeanExpression ?? "",
+      geneExpression?.expressedCellCount ?? "",
+    ];
+  } else {
+    return [
+      tissueName,
+      name,
+      total_count,
+      getTissuePercentage(geneExpression),
+      compareValueName,
+      geneName,
+      geneExpression?.meanExpression ?? "",
+      geneExpression?.scaledMeanExpression ?? "",
+      geneExpression?.expressedCellCount ?? "",
+    ];
+  }
+}
+
+function getTissuePercentage(geneExpression: ChartFormat | undefined) {
+  return !geneExpression
```

Comment:
maybe guard clause instead of ternary?

e.g.,

```ts
if (!geneExpression) return ""

return Number((geneExpression.tissuePercentage || 0) * 100).toFixed(2) + "%";
```